pom.xml

```diff
@@ -212,6 +212,7 @@
 							<failOnViolation>true</failOnViolation>
 							<violationSeverity>warning</violationSeverity>
 							<includeTestSourceDirectory>true</includeTestSourceDirectory>
+							<configLocation>${checkstyle.config.location}</configLocation>
```

Comment:
So strange.  Look at the comment in line 202 above, not sure what the rationale is to keep `./spring-cloud-dataflow-server-local/pom.xml ` in sync w/ changes to this plugin.

Reply:
Because in this case the Spring-cloud-dataflow-server-local uses the `spring-boot-starter-parent` as its parent.